packages/glibc.rb

```diff
@@ -41,19 +42,22 @@ class Glibc < Package
        x86_64: '5fe94642dbbf900d22b715021c73ac1a601b81517f0da1e7413f0af8fbea7997'
     })
   when '2.32' # All architectures with updates past M92.
+  version '2.32-1'
+    # Currently just using the binaries for 2.27, since the 2.32 
+    # glibc is causing segfaults in locally compiled binaries.
     source_url 'https://ftpmirror.gnu.org/glibc/glibc-2.32.tar.xz'
     source_sha256 '1627ea54f5a1a8467032563393e0901077626dc66f37f10ee6363bb722222836'
 
-  binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.32_armv7l/glibc-2.32-chromeos-armv7l.tpxz',
-    armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.32_armv7l/glibc-2.32-chromeos-armv7l.tpxz',
-    x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.32_x86_64/glibc-2.32-chromeos-x86_64.tpxz'
-  })
-  binary_sha256({
-    aarch64: '1fb970e490c988e49672e3e5d1edcaea795323e4fbb74559dea3bd35d7e6c63c',
-    armv7l: '1fb970e490c988e49672e3e5d1edcaea795323e4fbb74559dea3bd35d7e6c63c',
-    x86_64: '794745d17eae977dccf6d5635f67e5bb1465286a3888bc5e69eda487e87654ae'
-  })
+    binary_url({
+      aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_armv7l/glibc-2.27-chromeos-armv7l.tar.xz',
+       armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_armv7l/glibc-2.27-chromeos-armv7l.tar.xz',
+       x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/glibc/2.27_x86_64/glibc-2.27-chromeos-x86_64.tar.xz'
```

Comment:
We're changing from 2.32 to 2.27?  Was this intended?

Reply:
Yes. The crt files used for compiles in the globe 2.27 package don't appear to pull in `libgcc-s.so.1` as a dependency.